### PR TITLE
Fix release as agreed by TSC

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

File: docs/GE_Requirements.md [hunk 1]
```diff
@@ -148,23 +148,41 @@ contributions **MUST** include clauses as follows:
...
-    FIWARE releases evolve.
+    TSC level. Milestones will be aligned with the celebration of FIWARE Summits
+    by default. Additional ones can be added if strictly needed and agreed at
+    TSC level.
+
+### GE Releases
 
-The release note for each versioned release should include a list of the latest
-version of interoperable enablers it has been tested against as follows:
+-   Everytime there is a FIWARE Release, GE Owners **MUST** tag the GE release
```
Comment (jason-fox) [inline docs/GE_Requirements.md]: Typo - Every time

File: docs/GE_Requirements.md [hunk 2]
```diff
@@ -148,23 +148,41 @@ contributions **MUST** include clauses as follows:
...
+    pertaining to such FIWARE Release with `FIWARE<major>.<minor>`. Please note
+    that this imply that certain GE releases could be double tagged. The
+    referred tag will be intended to mark clearly that such GE Release is part
+    of and compatible with the corresponding FIWARE Release. Docker images or
+    other associated artefacts **MUST** also be properly tagged with the
+    referred FIWARE tag.
+
+"Compatibility" with a release means compatibility with the FIWARE Context
+Broker in that release, i.e. the FIWARE GE owner commits to do the relevant
+testing.
```
Comment (jason-fox) [inline docs/GE_Requirements.md]: To align this with the timetabling PR, this should be explicitly:

> relevant functional and integration testing

File: docs/GE_Requirements.md [hunk 3]
```diff
@@ -148,23 +148,41 @@ contributions **MUST** include clauses as follows:
...
-version of interoperable enablers it has been tested against as follows:
+-   Everytime there is a FIWARE Release, GE Owners **MUST** tag the GE release
+    pertaining to such FIWARE Release with `FIWARE<major>.<minor>`. Please note
+    that this imply that certain GE releases could be double tagged. The
+    referred tag will be intended to mark clearly that such GE Release is part
+    of and compatible with the corresponding FIWARE Release. Docker images or
+    other associated artefacts **MUST** also be properly tagged with the
+    referred FIWARE tag.
+
+"Compatibility" with a release means compatibility with the FIWARE Context
```
Comment (jason-fox) [inline docs/GE_Requirements.md]: This implies that likely breaking changes within the Context Broker should be scheduled and announced beforehand (or more like roadmapped) to give GEs a chance to catch up.

For example removing the NGSI v1 interface could break compatibility so dev teams would need to be warned that they may have work to do.

Does this need to be added to the text somehow?

File: docs/GE_Requirements.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ contributions **MUST** include clauses as follows:
 A "FIWARE Release" shall contain versioned releases of all Generic Enablers
 found within the [catalogue](https://www.fiware.org/developers/catalogue/). All
 GEs pertaining to a FIWARE Release must work seamlessly with the corresponding
-FIWARE Context Broker release.
+FIWARE Context Broker part of such FIWARE Release.
```
Comment (jason-fox) [inline docs/GE_Requirements.md]: > ... with the corresponding FIWARE Context Broker part of such ...
=> 
> with the version of the  FIWARE Context Broker corresponding to that FIWARE Release

Comment (jmcanterafonseca) [reply docs/GE_Requirements.md]: ok

File: docs/GE_Requirements.md (hunk 3 above)
Comment (jmcanterafonseca) [reply docs/GE_Requirements.md]: added

File: docs/GE_Requirements.md (hunk 2 above)
Comment (jmcanterafonseca) [reply docs/GE_Requirements.md]: yep

**Pull request merged**